src/commands/a32nx/freetext.ts

```diff
@@ -0,0 +1,13 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const freetext: CommandDefinition = {
+    name: ['freetext', 'ft'],
+    description: 'Provides a link to the FlyByWire free text tutorial video',
+    category: CommandCategory.A32NX,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | Free Text',
+        description: 'Please see our [guide](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/freetext) on how to use the FlyByWire A32NX Free Text feature.',
```

Comment:
Just to add the / to the link as not having it for some reason messes images on the page up? not sure why...

```suggestion
        description: 'Please see our [guide](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/freetext/) on how to use the FlyByWire A32NX Free Text feature.',
```